src/pages/basic_data/app_admin/central_import/CentralImportDuplicationCheckPage.tsx

```diff
@@ -74,6 +74,20 @@ export const CentralImportDuplicationCheckPage = () => {
               publication={registration}
               setDuplicateSearchFilters={setDuplicateSearchFilters}
             />
+            <Button
+              variant="outlined"
+              color="primary"
+              component={Link}
+              to={`${getImportCandidatePagePath(identifier)}`}>
```

Comment:
`getImportCandidatePagePath` returnerer uansett en streng, så trenger ikke å wrappe en streng rundt der igjen
```suggestion
              to={getImportCandidatePagePath(identifier)}>
```